config/initializers/pluck_in_batches.rb

```diff
@@ -27,7 +27,7 @@ def pluck_in_batches(columns, batch_size: 1_000)
       batch = self.limit(batch_size).offset(offset)
       items = batch.pluck(*columns)
       page += 1
-      yield items
+      yield items if items.present?
```

Comment:
What happens if items aren't present?  Seems odd we haven't hit this yet, but _maybe_ we've never plucked in batches with no results?

Reply:
I guess we haven't -- when I ran into it, it invoked the block on an empty array, which seemed wrong.